ocs_ci/deployment/deployment.py

```diff
@@ -247,3 +241,24 @@ def add_node(self):
         Implement platform specif add_node in child class
         """
         raise NotImplementedError("add node functionality node implemented")
+
+    def patch_default_sc_to_non_default(self):
+        """
+        Patch storage class which comes as default with installation to non-default
+        """
+        sc_to_patch = None
+        platform = config.ENV_DATA.get('platform')
```

Comment:
it's already defined in constructor https://github.com/red-hat-storage/ocs-ci/blob/56b376da01327b01e96633ea011ffe653fd09a1a/ocs_ci/deployment/deployment.py#L29